resources/pt/cemu.po

```diff
@@ -385,55 +552,78 @@ msgstr ""
 "Os idiomas disponíveis estão armazenados na pasta de traduções\n"
 "Será necessário reiniciar o emulador para a alteração fazer efeito"
 
-#: GeneralSettings2.cpp:83
+#: GeneralSettings2.cpp:92
 msgid "Remember main window position"
 msgstr "Lembrar posição da janela principal"
 
-#: GeneralSettings2.cpp:85
+#: GeneralSettings2.cpp:94
 msgid "Restores the last known window position and size when starting Cemu"
 msgstr ""
 "Restaura a última posição e tamanho conhecidos da janela ao inicializar o "
 "Cemu"
 
-#: GeneralSettings2.cpp:88
+#: GeneralSettings2.cpp:97
 msgid "Remember pad window position"
 msgstr "Lembrar posição da janela de controles"
 
-#: GeneralSettings2.cpp:90
+#: GeneralSettings2.cpp:99
 msgid "Restores the last known pad window position and size when opening it"
 msgstr ""
 "Restaura a última posição e tamanho conhecidos da janela de configuração de "
 "controles ao abri-la"
 
-#: GeneralSettings2.cpp:94
+#: GeneralSettings2.cpp:103
 msgid "Discord Presence"
 msgstr "Presença no Discord"
 
-#: GeneralSettings2.cpp:96
+#: GeneralSettings2.cpp:105
 msgid ""
 "Enables the Discord Rich Presence feature\n"
 "You will also need to enable it in the Discord settings itself!"
 msgstr ""
 "Ativa a funcionalidade do Discord Rich Presence\n"
 "Também será necessário deixá-la ativada nas configurações do Discord"
 
-#: GeneralSettings2.cpp:101
+#: GeneralSettings2.cpp:110
 msgid "Fullscreen menu bar"
 msgstr "Barra de menus em tela cheia"
 
-#: GeneralSettings2.cpp:103
+#: GeneralSettings2.cpp:112
 msgid ""
 "Displays the menu bar when Cemu is running in fullscreen mode and the mouse "
 "cursor is moved to the top"
 msgstr ""
 "Mostra a barra de menu quando o Cemu estiver em modo tela cheia e o cursor "
 "do mouse for movido para a parte superior da tela"
 
-#: GeneralSettings2.cpp:113
+#: GeneralSettings2.cpp:115
+msgid "Automatically check for updates"
+msgstr "Verificar autom. por atualizações."
```

Comment:
This is a description near a checkbox, right? For UI consistency it should not end with a dot.
If I remember wrong, just ignore this comment.